src/constants/bestiary.js

```diff
@@ -541,12 +541,12 @@ export const BESTIARY = {
         name: "Pigman",
```

Comment:
This looks wrong? They removed the Pigman Besitary

Reply:
get a life.

Ik I'm waiting for u to update constants, I'm using https://github.com/NotEnoughUpdates/NotEnoughUpdates-REPO/blob/prerelease/constants/bestiary.json (old constants were broken I missed smth)

Reply:
> get a life.
What? 

I already sent you a version without it?

Reply:
> > get a life.
> 
> What?

You responded liek 10 mins after I created the PR :skull: 

> I already sent you a version without it?

Yeah but it wasn't ifnal version so I ditched it